pt/security/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: page
+title: "Segurança"
+lang: pt
+---
+
+Aqui você encontrará informações sobre problemas de segurança do Ruby.
+{: .summary}
+
+## Reportando Vulnerabilidades de Segurança
+
+Vulnerabilidades de segurança devem ser reportadas por e-mail
+para o endereço security@ruby-lang.org ([a chave pública PGP](/security.asc)),
+que é uma lista de e-mails privada. Os problemas reportados serão
+publicados após as correções.
+
+Os membros da lista de e-mails são pessoas que mantém o Ruby
+(committers do Ruby e autores de outras implementações do Ruby,
+distribuidores, e plataformas PaaS). Os membros precisam ser
+pessoas individuais, outras listas de e-mail não são permitidas.
+
+## Problemas conhecidos
+
+Estes são os problemas recentes:
+
+{% include security_posts.html %}
+
+Outros problemas conhecidos:
```

Comment:
This list to untranslated posts should be dropped and replaced with:

  `See [the English page](/en/security/) for prior security related posts.` (translated)